### PR TITLE
4. View a list of all reviews

### PR DESCRIPTION
**Pull request by MartinSWDev**
Load items onto Home page with basic data and css

### CODE REVIEW

File: src/components/Displays/DisplayAll.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import uuid from 'react-uuid';
+
+const DisplayAll = () => {
+  const [allReviews, setAllReviews] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get('https://martinswdev-be-nc-games.herokuapp.com/api/reviews')
+      .then(({ data }) => {
+        console.log(data.reviews);
```
Comment (yalicee) [inline src/components/Displays/DisplayAll.jsx]: do you need this one?

File: src/components/Displays/DisplayAll.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      {allReviews.map((item) => {
+        return (
+          <div key={uuid()} className="displayAll-review">
+            <img
+              key={uuid()}
+              src={item.review_img_url}
+              alt={item.title}
+              className="displayAll-review__img"
+            />
+            <h2 key={uuid()}>{item.title}</h2>
```
Comment (yalicee) [inline src/components/Displays/DisplayAll.jsx]: I don't think you need all the keys and uuiids. Just one will do

**Review by yalicee — APPROVED** [review]
Looks good :)

File: src/components/Displays/DisplayAll.jsx (hunk 1 above)
Comment (MartinSWDev) [reply src/components/Displays/DisplayAll.jsx]: nope I missed that one!

File: src/components/Displays/DisplayAll.jsx (hunk 2 above)
Comment (MartinSWDev) [reply src/components/Displays/DisplayAll.jsx]: Was having trouble when they were lists, will remove them and see what errors show up

**Pull request merged**